Rakefile

```diff
@@ -1,24 +1,19 @@
 require 'rake'
 require 'rake/testtask'
 
+require_relative 'lib/tasks/exercise'
+require_relative 'lib/tasks/exercise_test_tasks'
 require_relative 'lib/tasks/exercise_tests_runner'
```

Comment:
`lib/tasks/exercise_test_tasks` is the only thing that should be included here.
The other includes belong inside that file.